core/database_arango.py

```diff
@@ -670,6 +670,14 @@ def filter(
                 conditions.append(f"COUNT(FOR c IN o.context[*] FILTER REGEX_TEST(c.@arg{i}_key, @arg{i}_value) RETURN c) > 0")
                 aql_args[f'arg{i}_key'] = context_field
                 sorts.append(f"o.context[*].@arg{i}_key")
+            elif key == 'created':
+                operator = value[0]
+                if operator not in ['<', '>']:
+                    operator = '='
```

Comment:
Does it handle `<=` or `>=`?

Reply:
Not in this case, it doesn't. I figure that for datetimes it's not that critical, since frmo what I tested it still goes down to the second on both sides to do the comparison (so `>2020-01-01` will give you everything after `2020-01-01T00:00:00`, which will also include 2020-01-01. You can fill in whatever kind of time you want as long as it's ISO-ish